pkg/types/aws/platform.go

```diff
@@ -106,6 +106,12 @@ type Platform struct {
 	// +optional
 	LBType configv1.AWSLBType `json:"lbType,omitempty"`
 
+	// eipAllocations contains Elastic IP (EIP) allocations for AWS resources
```

Comment:
gofmt will probably want this to be 
```suggestion
	// EIPAllocations contains Elastic IP (EIP) allocations for AWS resources
```